batch_job/src/Makefile

```diff
@@ -43,6 +44,12 @@ work_queue_factory: work_queue_factory.o libbatch_job.a $(EXTERNAL_LIBRARIES)
 work_queue_pool: work_queue_factory
 	cp $< $@
 
+makeflow_amazon_batch_cleanup:
+	cp amazon_batch_cleanup_script.sh makeflow_amazon_batch_cleanup
```

Comment:
Let's rename amazon_batch_cleanup_script.sh and amazon_batch_script.sh  to the names without .sh. That way there won't be confusion in the future that they are the same files.